File: Dec2020/6Dec/RichestCustomerWealth.cpp
/*
Problem name: 1672. Richest Customer Wealth
Problem link: https://leetcode.com/problems/richest-customer-wealth/
*/

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int max=-1, sum=0;
        for(int i=0;i<accounts.size();++i){
            sum=0;
            for(int j=0;j<accounts[i].size();++j){
                sum+=accounts[i][j];
            }
            if(sum > max)
                max = sum;
        }
        return max;
    }
};

File: Dec2020/6Dec/RichestCustomerWealth.java
/*
Problem Name: 1672. Richest Customer Wealth
Problem Link: https://leetcode.com/problems/richest-customer-wealth/
*/
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0, sum, n = accounts.length, m = accounts[0].length;
        for(int i=0;i<n;i++){
            sum = 0;
            for(int j=0;j<m;j++)
                sum += accounts[i][j];
            max = Math.max(max, sum);
        }
        return max;
    }
}
